app/models/application_record.rb

```diff
@@ -1,7 +1,16 @@
 require_relative 'concerns/audit_associations'
+require_relative 'concerns/host_mix'
+require_relative 'concerns/has_many_common'
+require_relative 'concerns/strip_whitespace'
+require_relative 'concerns/parameterizable'
```

Comment:
`concerns` subfolders are in a autoload path, so you should not need this, it should work by leaving it out to autoloader to load those. Being in autoload_path means they are roots and `concerns/host_mix` => `HostMix` is expected by autoloader.

I believe the previous require is here by mistake and can be removed while we are at it.